src/test/java/service/FIleBookServiceTest.java

```diff
@@ -0,0 +1,95 @@
+package service;
+
+import model.Book;
+import model.Status;
+import org.junit.jupiter.api.*;
+import repository.FileRepository;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class FIleBookServiceTest {
+    private BookService bookService;
+
+    @BeforeEach
+    void setUp() {
+        FileRepository fileRepository = new FileRepository("test.csv");
+        bookService = new BookService(fileRepository);
+        fileRepository.saveBook(new Book(fileRepository.createBookNo(), "대여 가능 책", "작가1", 123, Status.AVAILABLE));
+        fileRepository.saveBook(new Book(fileRepository.createBookNo(), "대여 중인 책", "작가2", 123, Status.BORROWED));
+        fileRepository.saveBook(new Book(fileRepository.createBookNo(), "분실된 책", "작가3", 123, Status.LOST));
+    }
+
+    @Test
+    @DisplayName("도서 등록 테스트")
+    void save() {
+        bookService.saveBook("해리포터", "J.K 롤링", 255);
+
+        assertThat(bookService.findAllBook().size()).isEqualTo(4);
```

Comment:
assertJ에는 여러가지 메서드들이 있습니다. 이런 경우엔 

```suggestion
        assertThat(bookService.findAllBook()).hasSize(4);
```
이런식으로도 사용할수 잇는데요. 어떤게 더 좋을지는 한번 생각해보세요!